onnx_tf/handlers/frontend/onehot.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+from onnx.helper import make_node
+from onnx.helper import make_tensor
+
+from onnx_tf.common.data_type import any_dtype_to_onnx_dtype
+from onnx_tf.common import exception
+from onnx_tf.common import get_unique_suffix
+from onnx_tf.handlers.frontend_handler import FrontendHandler
+from onnx_tf.handlers.handler import onnx_op
+from onnx_tf.handlers.handler import tf_op
+
+
+@onnx_op("OneHot")
+@tf_op("OneHot")
+class OneHot(FrontendHandler):
+
+  @classmethod
+  def args_check(cls, node, **kwargs):
+    if node.inputs[2] not in kwargs["consts"]:
+      exception.CONST_NOT_FOUND_EXCEPT(node.inputs[2], node.op_type)
+    if node.inputs[3] not in kwargs["consts"]:
+      exception.CONST_NOT_FOUND_EXCEPT(node.inputs[3], node.op_type)
+
+  @classmethod
+  def version_9(cls, node, **kwargs):
+    indices = node.inputs[0]
+    depth = node.inputs[1]
+    axis = node.attr.get('axis', -1)
+
+    on_value = kwargs['consts'][node.inputs[2]].item(0)
+    off_value = kwargs['consts'][node.inputs[3]].item(0)
+    values = np.array([off_value, on_value])
+    constant_output_name = node.outputs[0] + '_' + get_unique_suffix()
+
+    constant_node = make_node(
+        'Constant',
+        inputs=[],
+        outputs=[constant_output_name],
+        value=make_tensor(
```

Comment:
Could use `numpy_helper.from_array` to simplify code.

Reply:
@fumihwh updated, may you please review again.